langium/er-dsl/src/language/MetaModel/MultiRelationship.ts

```diff
@@ -55,17 +55,45 @@ export class MultiRelationship {
     public toString(): string {
         return this.simpleString();
     }
+    
+    //parse cardinality * -> N 
+    aggregateCardinality(cardinality: Cardinality): string {
+        if (cardinality == "*"){
+            return "N";
+        }
+        return cardinality.toString();
+    }
+
+    public toPlantUML(): string {
+        let result: string = '';
+
+        result += `relationship "${this.name}" as ${this.identifier} { }\n`
 
+        for (const connection of this.connections) {
+            const { entity, lower_cardinality, upper_cardinality, identifies } = connection;
+            let cardinality = lower_cardinality === upper_cardinality 
+            ? lower_cardinality
+            : `${lower_cardinality}..${upper_cardinality}`;
```

Comment:
For readability i would like to break the line elsewhere. I have not tested the suggestion. 
It does not look like the variable is changed afterwards, so `const` is a better choice
```suggestion
            const cardinality = (lower_cardinality === upper_cardinality) ? lower_cardinality
            : `${lower_cardinality}..${upper_cardinality}`;
```

Or putting everything in one line, or making the ranged earlier like:
```ts
const ranged = `${lower_cardinality}..${upper_cardinality}`
const cardinality = (lower_cardinality === upper_cardinality) ? lower_cardinality : ranged
```